packages/bridge/src/webpack/esbuild.ts

```diff
@@ -0,0 +1,56 @@
+import { EsbuildPlugin } from 'esbuild-loader'
+import type { Configuration } from 'webpack'
+import type { Nuxt } from '@nuxt/schema'
+
+export interface WebpackConfigContext {
+  nuxt: Nuxt
+  config: Configuration
+  isServer: boolean
+  transpile: RegExp[]
+}
+
+export function esbuild (ctx: WebpackConfigContext) {
```

Comment:
Based on the following files:
https://github.com/nuxt/nuxt/blob/main/packages/webpack/src/presets/esbuild.ts